mobile/test/common/integration/test_server.h

```diff
@@ -50,6 +55,11 @@ class TestServer {
    * Returns the port that got attributed. Can only be called once the server has been started.
    */
   int getServerPort();
+
+  /**
+   * Sets headers and data for the test server to return on all future requests.
+   */
+  void setHeadersAndData(std::string header_key, std::string header_value, std::string data);
```

Comment:
these should either be const std::string&, or the arguments should be std::move'd in the setHeadersAndData function body

Reply:
It looks like these are still passed by value and not moved. Let's just make them string_view?

Reply:
switched to string_view